### PR TITLE
fix: typo, grammar issues in documentation

### PR DESCRIPTION
**Pull request by dxphilo**
This PR should provide a fix for #6567

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -232,11 +232,11 @@ For crafting much better culture and Developer Experience, we reccomend some ext
 - [Refined Github](https://chrome.google.com/webstore/detail/refined-github/hlepfoohegkhhmjieoechaddaejaokhf)
 - [Github HoverCard](https://chrome.google.com/webstore/detail/github-hovercard/mmoahbbnojgkclgceahhakhnccimnplk)
 - [Gifs for Github](https://chrome.google.com/webstore/detail/gifs-for-github/dkgjnpbipbdaoaadbdhpiokaemhlphep)
-- Feel free add your extension which helps you organize and being productive on Github
+- Feel free to add your extension, which helps you organize and be productive on GitHub.
 
 ### Does it follow the contributor covenant
 
-This repository has a [code of conduct](CODE_OF_CONDUCT.md). This repository has a code of conduct, and I will remove things that do not respect it.
+This repository has a [code of conduct](CODE_OF_CONDUCT.md), and we will remove things that do not respect it.
```
Comment (roiLeo) [inline CONTRIBUTING.md:239]: I would remove comma before `and`, or remove `and` and keep comma

**Review by roiLeo — APPROVED** [review]
otherwise lgtm

File: CONTRIBUTING.md (hunk 1 above)
Comment (dxphilo) [reply CONTRIBUTING.md]: Absolutely, I'll just remove the comma for now.

**Comment by yangwao** [comment]
ok, but next time PR template best

**Pull request merged**